### PR TITLE
Add rotation from direction and up vector matrix

### PR DESCRIPTION
**Pull request by MortimerGoro**

### CODE REVIEW

File: include/vrb/Matrix.h [hunk 1]
```diff
@@ -91,6 +91,25 @@ class Matrix {
     return result;
   }
 
+  static Matrix Rotation(const Vector& aDirection, const Vector& aUp = vrb::Vector(0.0f, 1.0f, 0.0f)) {
```
Comment (bluemarvin) [inline include/vrb/Matrix.h]: I think you want to normalize the aDirection to be safe?

Comment (MortimerGoro) [reply include/vrb/Matrix.h]: yes, it seems safer. I updated the code.

File: include/vrb/Matrix.h [hunk 2]
```diff
@@ -91,6 +91,26 @@ class Matrix {
...
+    Matrix result;
+    const vrb::Vector dir = aDirection.Normalize();
```
Comment (bluemarvin) [inline include/vrb/Matrix.h]: You need to then use the dir var bellow when calculating the result as well I assume?

Comment (MortimerGoro) [reply include/vrb/Matrix.h]: Yes, forgot about it. PR updated

**Pull request merged**